Documentation/ApiOverview/Events/EventDispatcher/_ServicesOverrideCustom.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyVendor\MyExtension\EventListener;
+
+use TYPO3\CMS\Core\Attribute\AsEventListener;
+use TYPO3\CMS\Frontend\Event\ModifyHrefLangTagsEvent;
+
+// Important: Use the 'identifier' of the original event here to be replaced!
+#[AsEventListener(
```

Comment:
Again, this has to be adjusted when backporting. Make it the way 12.4 needs it and it the changes in a Post-PR only applied to main.

Reply:
The PHP attribute is forwards compatible and this example should run in v12. I've added a note.

I really don't want this to divert into different implementations then, because also the main description would need adaptation because for v12 you need both the PHP and the YAML file.

Reply:
The PHP attribute AsEventListener is a feature in v13. Perhaps you mix that up with the AsCommand attribute, which was a task and backported to v12 (which I don't understand why).